src/services/OParl/consultation.test.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  basicImportTest,
+  invalidUrlTest,
+  setupTestDataBase,
+} from '../../helpers';
+import { importConsultation } from './consultation';
+
+const helperFuncs = setupTestDataBase('test_consultation_import');
+
+// TODO: change test url to something within our control, that uses all fields
+const testUrl =
+  'https://www.muenchen-transparent.de/oparl/v1.0/consultation/48823';
```

Comment:
i see that some tests need and url and others not. and they differ between tests, right? so we could think about a concept where to put these urls with different paths when we a host within our control.

Reply:
for now they should all have todo comments so we can find them easily.

we could centralize them somewhere, but i do not see a great benefit.

the tests that need urls are the tests for the importers (services/OParl/...), and if we always put them on top of the files, they are always close to the location where they are exclusively used.

do you see any benefits for extracting them?

Reply:
i would see benefits in extracting the host and main path like `https://www.muenchen-transparent.de` and `/oparl/v1.0/` for example somewhere and use it everywhere needed. this way it can be changed on one place instead of searching and replacing multiple ones.

Reply:
hmm. the thing is if we want to interchange the oparl systems we would very likely need to change all the url suffixes anyways, because they will not exist/be the same for the new system. 🤔